tests/wpt/mozilla/tests/mozilla/table_rowspan_colspan_crashtest.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<!-- Shouldn't crash (#20162) -->
+<table border>
+    <tr><td>a</td><td>b</td><td>c</td><td>d</td></tr>
+    <tr><td rowspan=2>a</td><td>b</td><td colspan=2 rowspan=2>cd</td>  </tr>
+    <tr>                    <td>b</td></tr>
```

Comment:
Maybe upstream it to wpt? Maybe not too worth it...

Reply:
Probably not, this is a pretty specific crash